packages/fuels-accounts/src/provider/retryable_client.rs

```diff
@@ -357,4 +376,24 @@ mod custom_queries {
         #[arguments(id: $transaction_id)]
         pub transaction: Option<TransactionIdFragment>,
     }
+
+    #[derive(cynic::QueryFragment, Clone, Debug)]
+    #[cynic(
+        schema_path = "./target/fuel-core-client-schema.sdl",
+        graphql_type = "Contract"
+    )]
+    pub struct ContractIdOnly {
```

Comment:
fuel core client already has a `ContractIdFragment`, you can use that. We use it in the blob query.